### PR TITLE
xGov-7 Livestream Integration

### PR DESCRIPTION
**Pull request by CasiniNift**

### CODE REVIEW

**Comment by averagezen** [comment]
Curious how many current active content creators and users on Niftgen and how many we can estimate would be added to the ecosystem with this proposal?

**Comment by CasiniNift** [comment]
Hi @averagezen 
We have onboarded 12 content creators. Not all of them have started posting content. In terms of users, over 500 people have created accounts. Estimating how many users would be added to the ecosystem is hard. A lot will depend on how often the creators will do live streams.

File: Proposals/2 [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+period: 7
```
Comment (SudoWeezy) [inline Proposals/2:2]:
```suggestion
period: 7
```

File: Proposals/2 [hunk 2]
```diff
@@ -0,0 +1,103 @@
+---
+period: 7 
+title: Livestream Integration 
+author: Christian Casini ( @Niftgen )
```
Comment (SudoWeezy) [inline Proposals/2:4]:
```suggestion
author: Christian Casini (@Niftgen)
```

File: Proposals/2 [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+company_name:Niftgen Inc  
+category: dApp
```
Comment (SudoWeezy) [inline Proposals/2:6]:
```suggestion
category: dApps
```

File: Proposals/2 [hunk 4]
```diff
@@ -0,0 +1,103 @@
+---
+period: 7 
+title: Livestream Integration 
+author: Christian Casini ( @Niftgen )
+company_name:Niftgen Inc  
+category: dApp 
+focus_area: SocialFi, web 3 social, and NFTs
```
Comment (SudoWeezy) [inline Proposals/2:7]: must be one of: `Banking`, `Defi`, `Dex`, `Gaming`, `Identity`, `Marketplace`, `Metaverse`, `NFT`, `Oracle`, `Storage`, `User Onboarding`, `IDE`, `Teal`, `Deployment`, `Libraries`, `Monitoring`, `Node`, `Education`

File: Proposals/2 [hunk 5]
```diff
@@ -0,0 +1,103 @@
+---
+period: 7 
+title: Livestream Integration 
+author: Christian Casini ( @Niftgen )
+company_name:Niftgen Inc  
+category: dApp 
+focus_area: SocialFi, web 3 social, and NFTs 
+open_source: No
+amount_requested: 192,000 ALGOs
```
Comment (SudoWeezy) [inline Proposals/2:9]:
```suggestion
amount_requested: 192000
```

**Review by SudoWeezy — CHANGES_REQUESTED** [review]
Thanks for your proposal @CasiniNift. I have suggested some formatting changes to have a proper structure.

File: Proposals/2 (hunk 4 above)
Comment (CasiniNift) [reply Proposals/2:7]: @SudoWeezy Ok, the closest thing would be User Onboarding. I think SocialFi should be added as a category.

**Comment by No-Cash-7970** [comment]
It would be helpful if you rename the file to `xgov-7.md` so GitHub can show a human friendly version of the proposal you wrote.

**Comment by CasiniNift** [comment]
> It would be helpful if you rename the file to `xgov-7.md` so GitHub can show a human friendly version of the proposal you wrote.

Sure, how would I do that exactly ? 
Not supper familiar with how GitHub works

**Comment by CasiniNift** [comment]
> > > It would be helpful if you rename the file to `xgov-7.md` so GitHub can show a human friendly version of the proposal you wrote.
> > 
> > 
> > Sure, how would I do that exactly ? Not supper familiar with how GitHub works
> 
> You can click on the Edit file here:
> 
> image

@SudoWeezy done

File: Proposals/2 (hunk 4 above)
Comment (SudoWeezy) [reply Proposals/2:7]: New discussion was created here 
https://github.com/algorandfoundation/ARCs/pull/200

**Comment by 2lobo** [comment]
why is algorand needed to make niftgen work? the only thing that you want to use seems to be POAPs which are fine but i dont see how that should people onboard onto Algorand

how do you think to compete against the big live streaming platforms? what can you offer creators that others like twitch, youtube, kick,..., cant? why should i as a user come and watch things on niftgen when i can go to a mature platform like youtube where apparently the same content will be livestreamed on?

in my honest opinion it feels like a web2 project that wants to be affiliated with web3 but doesnt really benefit from web3

**Comment by CasiniNift** [comment]
> why is algorand needed to make niftgen work? the only thing that you want to use seems to be POAPs which are fine but i dont see how that should people onboard onto Algorand
> 
> how do you think to compete against the big live streaming platforms? what can you offer creators that others like twitch, youtube, kick,..., cant? why should i as a user come and watch things on niftgen when i can go to a mature platform like youtube where apparently the same content will be livestreamed on?
> 
> in my honest opinion it feels like a web2 project that wants to be affiliated with web3 but doesnt really benefit from web3

Hi @2lobo 
Thank you for your feedback. 

Here is a bit more clarification:
The way Niftgen and the creators make money is through subscriptions. Those subscriptions are all on chain. Users are reward for their engagement with a utility token  ( no monetary value) that can used to access features. 

We are not telling creators to stop using Twitch or YouTube. Creators use YouTube / Twitch to get discovered, but then bring their followers over to Niftgen to access exclusive content. Patroon is a more accurate competitor. 

The way Niftgen benefits from web 3 is because users are actually rewarded for engaging with content. On web 2 video platforms, your are just a passive user.

File: Proposals/2 [hunk 6]
```diff
@@ -0,0 +1,103 @@
+---
+period: 7
+title: Livestream Integration
+author: Christian Casini (@Niftgen)
+company_name: Niftgen Inc
+category: dApps
+focus_area: SocialFi, web 3 social, and NFTs
```
Comment (SudoWeezy) [inline Proposals/2:7]:
```suggestion
focus_area: Social
```

**Review by SudoWeezy — CHANGES_REQUESTED** [review]

**Comment by CasiniNift** [comment]
Hello 
I am withdrawing the proposal and will close the pull request

**Pull request closed without merging**